### DISCUSSION TITLE
Option to Expanded or Show Calendar by default

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#669] [Feature Requests]
![image](https://github.com/publishpress/PublishPress-Future/assets/83811723/983e16eb-82a8-46a2-9d81-cea5067a3b1e)

I wonder if this is possible?

original request: https://wordpress.org/support/topic/default-collapse-calendar/

**Comment by DafDodger** [comment]
This would be very handy for us as screen real estate is limited on our system. If we need the calendar, rarely, we can manually expand it. Thank you for considering this.

**Comment by rizaardiyanto1412** [comment]
@DafDodger
Just wondering, on which condition you enable PublishPress Future but dont really need Calendar?

**Reply by DafDodger** [reply]
We post news/sports stories that have a 2 day shelf life. So I set the default expiry to 2 days. So all we have to do is enable PPF and it will do the rest.  We rarely have a need to go further out than that - but if we do we could expand the calendar as needed.  With the calendar expanded by default we lose a good 2" of space in our right column - so we have to scroll to get to the next widget. Hope that makes sense. :)

Thanks

**Reply by rizaardiyanto1412** [reply]
Thanks for sharing your use case.

Does this step help in your case or no?
1. Enable Auto-enable for new post
2. Add "2 days" on the below
![image](https://github.com/publishpress/PublishPress-Future/assets/83811723/8e4afb0b-5cf9-4f79-9b6a-83f88db97145)
3. Hide PublishPress Future metabox here:
![image](https://github.com/publishpress/PublishPress-Future/assets/83811723/9d8060d4-0f15-42a9-baa2-4b8e066cda48)

**Comment by DafDodger** [comment]
Yes, that is a good suggestion. However while a lot of our posts are as I described above there are other categories that are not. Our local and state stories are manually managed and may stay relevant for longer than the default. The folks that posts to those categories would not appreciate if their posts disappeared automatically. They don't deal with PPF like some of the other contributors. I would not want to complicate their routine with now having to set PPF. 
I know it's complicated. :)
If we can get this feature added that would be good - if not it's not a deal breaker. Thanks!

**Reply by rizaardiyanto1412** [reply]
No worries, thanks for sharing your use case

**Comment by andergmartins** [comment]
Thanks, guys for the contribution. I'm closing this since this feature was implemented on a previous release.

**Comment by DafDodger** [comment]
We are now on 3.4.0.1  and I still do not see an option for this in the settings. As of 3.4.0.1  we are still getting the full calendar as the default. Please direct me to the setting that is causing this to be "solved".

Thank you

**Reply by DafDodger** [reply]
Anyone still here? If this has indeed been implemented, please direct us to the setting or explain why we are still getting full calendar on activation of new post.

**Comment by DafDodger** [comment]
@rizaardiyanto1412 @andergmartins 
RIght - support is lacking here. Please re-open this ticket and respond to my question.

**Comment by DafDodger** [comment]
@rizaardiyanto1412 @andergmartins
Nice, you have a great day, too.

**Comment by andergmartins** [comment]
Hi @DafDodger sorry for the delay. I did miss the notification about your messages.

I mentioned it as implemented because after you close the calendar once, it should remain closed until you manually open it. But I see what you mean. There is no settings for that, which require manually closing the calendar every time it is opened. I will add this to our list, considering the implemented solution does not fully fix the issue.

**Reply by andergmartins** [reply]
For future need, here is the issue: https://github.com/publishpress/PublishPress-Future/issues/881

**Comment by DafDodger** [comment]
Thank you! :)
